src/main/kotlin/com/coder/gateway/sdk/CoderCLIManager.kt

```diff
@@ -179,6 +185,8 @@ class CoderCLIManager @JvmOverloads constructor(
             token,
             "--global-config",
             coderConfigPath.toString(),
+            "--header-command",
+            settings.headerCommand
```

Comment:
Do we need to _both_ set the environment variable and set the CLI flag?  I think either has the same effect, and of them I'd prefer the environment variable, since if the `headerCommand` is empty, it'll look strange on logs.

Reply:
removed, thanks 👍